OneCommunityGlobal/HighestGoodNetworkApp
# Haohui - Gives the user permission to manage individual user permissions.

# Description
Please include the exact bug/functionality description and a summary of the changes/ related issues. Please also include any other relevant motivation and context:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/78375725/20b58dab-a869-44ba-91de-147587525a60)

## Related PRS (if any):
This frontend PR is related to the backend PR # 532.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as owner user call user A
4. go to dashboard→ Other Links→ Permissions Management→ Manage User Permissions
5. input another user name, call user B
6. find the "Manage User Permissions", click the add button
7. scroll down and click submit button
8. log out and login as user B
9. go to dashboard→ Other Links→ Permissions Management→ Manage User Permissions
10. check the user B able to manage user permissions.

## Screenshots or videos of changes:
https://drive.google.com/file/d/1gEo7ixKdH1-o3ZnFts0N_L_Hsm73m34M/view?usp=sharing

## Note:
Include the information the reviewers need to know.
